app/src/sharedTest/java/org/oppia/android/app/profile/AddProfileActivityTest.kt

```diff
@@ -38,6 +40,8 @@ import androidx.test.ext.junit.runners.AndroidJUnit4
 import androidx.test.rule.ActivityTestRule
 import com.google.common.truth.Truth.assertThat
 import dagger.Component
+import javax.inject.Inject
+import javax.inject.Singleton
```

Comment:
These are not properly ordered, they should be at the bottom of the import list. I mentioned this in my last review.